Representor/HTTP/HTTPRequest.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  HTTPRequest.swift
+//  Representor
+//
+//  Created by Kyle Fuller on 30/01/2015.
+//  Copyright (c) 2015 Apiary. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+import URITemplate
+
+extension HTTPTransition {
+  var suggestedParameterEncoding:ParameterEncoding {
+    let encodingMapping = [
+      "application/json": ParameterEncoding.JSON,
```

Comment:
I think it would be better to allow user's to register their own mime types and what it corresponds too.

Although the RFC states to use `application/json` there are still many servers which use the incorrect type such as `text/javascript` and it would be best to leave this up to the user.